static/docs/command-reference/import.md

```diff
@@ -32,15 +33,16 @@ source <abbr>project</abbr>. Both HTTP and SSH protocols are supported for
 online repositories (e.g. `[user@]server:project.git`). `url` can also be a
 local file system path to an "offline" repository (in this case instead of
 downloading, DVC may copy the target data from the external source project or
-it's cache).
+its cache).
 
 The `path` argument of this command is used to specify the location of the data
-to be downloaded within the source project. It should point to a data file or
-directory tracked by that project – specified in one of the
-[DVC-files](/doc/user-guide/dvc-file-format) of the repository at `url`. (You
-will not find data files directly in the external Git repository.) The source
-project should have a default [DVC remote](/doc/command-reference/remote)
-configured, containing them.)
+to be downloaded within the source project. It should point to a file or
+directory tracked by that project. It might be an <abbr>output</abbr> – in that
+case it should be specified in one of the
+[DVC-files](/doc/user-guide/dvc-file-format) of the repository at `url` and
+might not be present in the repository itself. When importing an output, the
+source project should have a default [DVC remote](/doc/command-reference/remote)
+configured, containing it.
```

Comment:
Just copy the same paragraph in https://dvc.org/doc/command-reference/get instead, please.